src/index.js

```diff
@@ -59,21 +57,29 @@ const interpolateAmplitudes = (amplitudes, startIndex, startValue, endIndex, end
   return prior.concat(modified).concat(posterior)
 }
 
-const addBeeper = (beeperArray, buffer, rampDuration) => {
-  const newBeeper = createNewBeeper(buffer, rampDuration);
-  beeperArray[0].gainNode.gain.linearRampToValueAtTime(0, audioCtx.currentTime + rampDuration)
+const addBeeper = ({ beepers, buffer }, data, rampDuration) => {
+  const newBeeper = createBeeper(220, 0.3, data, rampDuration);
```

Comment:
shouldn't we be using the `FUNDAMENTAL` we defined? i'm not too picky cause my slider pr is gonna have to replace this either way